gooddata-server-oauth2-autoconfigure/src/main/kotlin/BearerTokenReactiveAuthenticationManagerResolver.kt

```diff
@@ -108,7 +138,79 @@ private class JwtAuthenticationManager(
                         withState("error")
                         withAuthenticationMethod(AUTH_METHOD)
                     }
-                    parseJwtException(ex, jwtToken)
+
+                    when (ex.cause?.cause) {
+                        is ParseException -> {
+                            auditClient.recordLoginFailure(
+                                orgId = organization.id,
+                                userId = "",
+                                source = sourceIp,
+                                sessionContextType = AuthMethod.JWT,
+                                sessionContextIdentifier = "",
+                                errorCode = "JWT_DECODE_ERROR",
+                                details = mapOf("errorMessage" to "JWT token could not be parsed")
+                            ).subscribe()
```

Comment:
As as above. rather than calling .subscribe() we should incorporate the call to the existing flow.

+ same of all other .subscribe() calls in this PR